pkg/apis/v1/ec2nodeclass_hash_test.go

```diff
@@ -111,25 +114,25 @@ var _ = Describe("Hash", func() {
 	// doesn't work well with unexported fields, like the ones that are present in resource.Quantity
 	It("should match static hash when updating blockDeviceMapping volumeSize", func() {
 		nodeClass.Spec.BlockDeviceMappings[0].EBS.VolumeSize = resource.NewScaledQuantity(10, resource.Giga)
-		Expect(nodeClass.Hash()).To(Equal("5906178522470964189"))
+		Expect(nodeClass.HashForRegion("us-west-2")).To(Equal("5906178522470964189"))
```

Comment:
Can we add test that will validate nodeclass hash for ISO region to specific hash values? This will be used to when the hash function changes in the future and be a mechanism for us to update the hash version